<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeDialog</class>
 <widget class="QDialog" name="ShapeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>671</width>
     <height>151</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="rectanglePage">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Ширина</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="rectWidthSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Длина</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="rectHeightSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="circlePage">
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Радиус</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="circleRadiusSpinBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="trianglePage">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>34</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Сторона А</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="triangleSideASpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Сторона B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="triangleSideBSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Сторона C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="triangleSideCSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="fillCheckBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Залить фигуру</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fillColorButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать цвет заливки </string>
   </property>
  </widget>
  <widget class="QPushButton" name="outlineColorButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать цвет обводки</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>166</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>192</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Тип фигуры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="shapeComboBox">
      <item>
       <property name="text">
        <string>Прямоугольник</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Круг</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Треугольник</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
